ui/.gitignore

```diff
@@ -19,3 +19,5 @@ yarn-error.log*
 *.njsproj
 *.sln
 *.sw*
+
+package-lock.json
```

Comment:
i guess good practice to include this on repo...

Reply:
put package-lock it back in